packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2684,8 +2682,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             //    issue than sending.
             // Please note that this does not change file format, so it can be disabled in the future if this
             // optimization no longer makes sense (for example, batch compression may make it less appealing).
-            if (this._flushMode === FlushMode.TurnBased && type === ContainerMessageType.Attach &&
-                this.mc.config.getBoolean("Fluid.ContainerRuntime.enableAttachOpReorder") === true) {
+            if (type === ContainerMessageType.Attach &&
```

Comment:
Why remove TurnBased check? Won't that have any affect?

Reply:
It should not matter anymore, as the code below will flush any pending batches (including 1 op) if we are not batching. Makes code more elegant, IMHO :)